akeneo-design-system/src/components/Pagination/Pagination.tsx

```diff
@@ -0,0 +1,181 @@
+import React, {FC} from 'react';
+import styled, {css} from 'styled-components';
+import {getColor, AkeneoThemedProps} from '../../theme';
+
+type PaginationProps = {
+  /**
+   * The current page number (starting à 1).
+   */
+  currentPage: number;
+
+  /**
+   * The total number of pages.
+   */
+  itemsTotal: number;
+
+  /**
+   * The number of items per page. Default to 25.
+   */
+  itemsPerPage?: number;
+
+  /**
+   * Handler called when a pagination item is clicked.
+   */
+  onClick: (page: number) => void;
+};
+
+const MAX_PAGINATION_ITEMS_WITHOUT_SEPARATOR = 4;
+const PAGINATION_SEPARATOR = '...';
```

Comment:
```suggestion
const PAGINATION_SEPARATOR = '…';
```

I think the three dot sign is advised here. Context: https://musescore.org/en/node/280844